OptimizelySDK/Event/Dispatcher/HttpClientEventDispatcher45.cs

```diff
@@ -72,10 +72,20 @@ static HttpClientEventDispatcher45()
         /// Async version of DispatchEvent
         /// This is a "Fire and Forget" option
         /// </summary>
-        public void DispatchEvent(LogEvent logEvent)
+        public void DispatchEventSilently(LogEvent logEvent)
         {
             Task.Run(() => DispatchEventAsync(logEvent));
         }
+
```

Comment:
The prime effect of the P.R. #52 proposal will be to change the body of DispatchEvent .
However, I don't understand the purpose of keeping DispatchEventSilently around.
The SDK code doesn't call it and HttpClientEventDispatcher45 isn't documented in
Optimizely's developer documentation.  Isn't DispatchEventSilently dead code?  Why
isn't this deleted?

Reply:
Because I was not sure whether there was a legit reason behind DispatchEvent hiding asynchronous nature of DispatchEventAsync. So I left and renamed the old code as DispatchEventSilently thinking maybe you want to preserve a async version of DispatchEvent and name it more obviously. But if there is no strong reason behind it, then I agree DispatchEventSilently is dead code and should be removed.